contrib/gorilla/mux/mux_test.go

```diff
@@ -199,6 +199,43 @@ func TestAnalyticsSettings(t *testing.T) {
 	})
 }
 
+func TestSkippingOption(t *testing.T) {
+	skippingFunc := func(req *http.Request) bool {
+		return req.URL.Path == "/skip"
+	}
+
+	tests := []struct {
+		url       string
+		spanCount int
+	}{
+		{
+			url:       "/skip",
+			spanCount: 0,
+		},
+		{
+			url:       "/200",
+			spanCount: 1,
+		},
+	}
+
```

Comment:
Can we remove this line please? I've only recently started to add some [Style guidelines](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines) (check "General Guidelines", point 3) to this repository and specifically tried to mention this part.

Reply:
Good style guideline. I think I understand what you were thinking of, are you refer to line 224 aswell?